modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/regions/JP.json

```diff
@@ -4,328 +4,328 @@
 			"ja_JP": "愛知県"
 		},
 		"name": "Aichi",
-		"regionCode": 23.0
+		"regionCode": 23
```

Comment:
@wanderlast I see inconsistency from the other codes added. I don't see these treated as strings and they're are not decimal.